src/main/webapp/WEB-INF/view/search.jsp

```diff
@@ -0,0 +1,44 @@
+<%@ page import="codeu.model.data.User" %>
+<%@ page import="java.util.List" %>
+<%
+List<User> users = (List<User>) request.getAttribute("users");
+String searchRequest = (String) request.getAttribute("searchRequest");
+%>
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Search</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <%@ include file="navbar.jsp" %>
+
+  <div id="container">
+    <h1>Results<a href="" style="float: right">&#8635;</a></h1>
+
+    <div id="results">
+      <% 
+      if (users == null) {
+      %>
+        <p>No results for: <%= searchRequest %></p>
+      <%
+      }
+      else {
+      %>
+        <p>Results for: <%= searchRequest %></p>
+      <%
+        for (User user : users) {
+      %>
+          <p><%= user.getName() %></p>
```

Comment:
How about we make this a link to the user's profile page?

Reply:
Will do, forgot to mention in the description that this was only to test the search function.